### PR TITLE
Implemented some more checking of ACNV file outputs

### LINKED ISSUE #252: Change ACS integration test to check for correctness of output.
Current test only checks that all output files are created.  Test should be updated once we have a better idea of the correct output from initial rounds of evaluation.

### PR DESCRIPTION
**Pull request by LeeTL1220**
Closes #252

### CODE REVIEW

File: src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java [hunk 1]
```diff
@@ -45,10 +49,31 @@ public void testAllelicCapSegger() {
...
+        for (final File outputFile: outputFileList) {
+
+            // Check that all files are files with a size greater than 0.
+            Assert.assertTrue(outputFile.isFile());
+            Assert.assertTrue(outputFile.length() > 0);
+            try {
+
+                //Check that all files have:
+                //  - at least two lines and all either start with "#" or contain at least one "\t"
+                //  - at least two lines with tab (header + 1 segment)
```
Comment (samuelklee) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java]: i would say "column header + 1 segment" instead to disambiguate

File: src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java [hunk 2]
```diff
@@ -45,10 +49,31 @@ public void testAllelicCapSegger() {
...
+                final List<String> outputLines = FileUtils.readLines(outputFile);
+                Assert.assertTrue(outputLines.size() > 2);
```
Comment (samuelklee) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java]: >=

File: src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java [hunk 3]
```diff
@@ -45,10 +49,31 @@ public void testAllelicCapSegger() {
...
+                Assert.assertEquals(outputLines.stream().filter(l -> l.contains("\t") || l.startsWith("#")).count(), outputLines.size());
+                Assert.assertTrue(outputLines.stream().filter(l -> l.split("\t").length > 2).count() > 2, "File: " + outputFile + " does not seem to have at least one segment and a header.");
```
Comment (samuelklee) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java]: probably OK to assume that header lines will not contain two tabs or more, but perhaps add `!l.startsWith("#")` to be safe?

**Comment by samuelklee** [comment]
back to @LeeTL1220, OK to merge after minor fixes!

File: src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java (hunk 1 above)
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java]: Done

File: src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java (hunk 2 above)
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java]: Done

File: src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java (hunk 3 above)
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCNVIntegrationTest.java]: Done

**Pull request merged**